### DISCUSSION TITLE
Add Fonts?

### DISCUSSION THREAD

**Discussion by onlycs** [gitpod-io/openvscode-server#200] [Q&A]
How would I like to add a font to the Terminal? I tried installing it from the docker deployment, as I would on any other Linux machine (and I even verified it was there using `fc-list`.) Changing the font-face in settings made the terminal font all weird (definitely NOT the one I wanted)

How would I add a font? is there a workaround if it is not a feature yet. I know that on cdr/code-server, you can edit some HTML to add a `@font-face`.

I also don't have access to the developer tools menu (blocked by admin).

**Comment by jeanp413** [comment]
@TheAlphaReturns custom terminal font is working as expected, I just tested it myself. Just install the font on the client machine (where your browser is installed) and then update `"terminal.integrated.fontFamily"` setting.
If it's still not working, what browser and font are you using?
> I also don't have access to the developer tools menu (blocked by admin).

Not following here, to open developer tools just press f12 or `ctrl+shift+I` in chrome

**Comment by onlycs** [comment]
I'm on a docker deployment on https://railway.app, using openvscode-releases

**Comment by akosyakov** [comment]
As far as I know browser should be aware of fonts, so you have to install fonts on local machine.
